package.json

```diff
@@ -9,9 +9,12 @@
     "dev": "webpack -d --watch",
     "server": "nodemon server/server.js",
     "seed-data": "node db/database.js",
-    "seed": "node db/gen10MData.js",
+    "seed-mongo": "node db/gen10MData.js",
+    "seed-pg": "node db/gen10MDataSQL.js",
     "build-prod": "./node_modules/.bin/webpack --config ./webpack.production.config.js",
-    "build": "webpack -p"
+    "build": "webpack -p",
+    "pg-start": "pg_ctl -D /usr/local/var/postgres start",
```

Comment:
This seems like a script that would be specific to the user so I would maybe consider making it a local alias instead.